source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
Looks like building two bundles is the only way to go. @andrew--r, do you mind implementing it? If you do mind, just tell me, I’ll do it on my own.

As one of the solutions, we could specify two entry points:
```js
entry: {
  'likely': './source/likely.js',
  'likely-commonjs': './source/index.js'
}
```

which will make Webpack build two different bundles, and update `output.filename` to match the bundle name:

```js
filename: '[name].js'
```

Or feel free to use any other solution if you find it better.